scalacheck/src/main/scala/pathy/scalacheck/RandomSeg.scala

```diff
@@ -28,9 +28,10 @@ private[scalacheck] object RandomSeg {
   implicit val randomSegArbitrary: Arbitrary[RandomSeg] =
     Arbitrary {
       Gen.nonEmptyListOf(Gen.frequency(
-        100 -> Arbitrary.arbitrary[Char],
+        100 -> Gen.alphaNumChar,
```

Comment:
@mossprescott You answered my numerous questions and I think this PR is ready to be merged. There is still something that itches me though with this particular change. I wonder if we could create a `Arbitrary[PrettySeg]` or maybe just stay like this but bring up a bit the relative frequency of the truly arbitrary `Char`. I am concerned we are likely to miss some corner cases with such a generator.

Also somewhat unrelated thought, I wonder if it would be nicer if we had `20 -> Gen.chooseOneOf(escapedChars)` where `escapedChars = List('.', '/')`?